src/patternfly/components/NotificationDrawer/examples/NotificationDrawer.md

```diff
@@ -139,6 +139,7 @@ cssPrefix: pf-c-notification-drawer
 | `.pf-m-warning` | `.pf-c-notification-drawer__list-item` | Modifies a notification list item for the warning state. |
 | `.pf-m-danger` | `.pf-c-notification-drawer__list-item` | Modifies a notification list item for the danger state. |
 | `.pf-m-success` | `.pf-c-notification-drawer__list-item` | Modifies a notification list item for the success state. |
+| `.pf-m-recommendation` | `.pf-c-notification-drawer__list-item` | Modifies a notification list item for the recommendation state. |
```

Comment:
Just want to verify this should be called `pf-m-recommendation` over `pf-m-default`?

Reply:
@megan-hall @mceledonia @mcarrano what are your thoughts here?

Reply:
Thanks for flagging this @mcoker . Even though 'recommendation' conforms to the OpenShift use case, we don't use that term elsewhere in PatternFly. We defined the teal alert state as 'default' that can be used for anything that the consumer wants. Can we call it 'pf-m-default'?

Reply:
updated